src/Core/SettingsChangesHistory.cpp

```diff
@@ -69,7 +69,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
         /// Note: please check if the key already exists to prevent duplicate entries.
         addSettingsChanges(settings_changes_history, "25.6",
         {
-
+            {"min_outstreams_per_resize_after_split", UInt64(-1), 24, "New setting."},
```

Comment:
In ClickHouse, we usually use `0` to indicate that a particular limit/threshold shouldn't be applied.